product_sticker/models/product_attribute.py

```diff
@@ -0,0 +1,25 @@
+from odoo import api, fields, models
+
+
+class ProductAttribute(models.Model):
+    _inherit = "product.attribute"
+
+    sticker_ids = fields.One2many(
+        comodel_name="product.sticker",
+        inverse_name="product_attribute_id",
+        string="Product Stickers",
+        domain=lambda s: s.env["product.sticker"]._build_sticker_domain_company(),
+    )
```

Comment:
You can still do this and benefit from the mixin approach, if you find it useful:

```suggestion
class ProductAttribute(models.Model):
    _name = "product.attribute"
    _inherit = ["product.attribute", "product.sticker.mixin"]
    
    sticker_ids = fields.One2many(
        inverse_name="product_attribute_id",
    )

class ProductStickerMixin(models.AbstractModel):
    _name "product.sticker.mixin"

    sticker_ids = fields.One2many(
        comodel_name="product.sticker",
        string="Product Stickers",
        domain=lambda s: s.env["product.sticker"]._build_sticker_domain_company(),
    )
```

It's just an example showing that it's simple, but of course the mixin should go in a separate file etc.